sdk/security/keyvault/internal/ci.keyvault.yml

```diff
@@ -0,0 +1,28 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
Looks like the name of this file will need to be changed so its pipeline doesn't collide with the old one.  That, or we repurpose the old one to be used here.

CC @benbp